qos/evm/state.go

```diff
@@ -1,27 +1,39 @@
 package evm
 
 import (
-	"fmt"
 	"sync"
 
 	"github.com/pokt-network/poktroll/pkg/polylog"
 
 	"github.com/buildwithgrove/path/protocol"
 )
 
+// EndpointStoreConfig captures the modifiable settings of the EndpointStore.
```

Comment:
The struct is called `serviceStateConfig` so I believe there was some copy-pasta.

Please revisit the whole comment.